README.md

```diff
@@ -1 +1,64 @@
-# Beyond-07-team-4
\ No newline at end of file
+# Job Seeker
+
+**Job Seeker** is a web platform, built for people to post and find jobs.
+It might serve students, freelancers or organzations.
+
+## How To Run The Project
+
+> NOTE: Depending on your OS, the commands in this README should be run with the following shells:
+> - *Windows:* on **cmd** or **Powershell**.
+> - *Linux:* on **terminal**.
+
```

Comment:
This section is not clear enough, 
if you have a different deployments on different OS then you better write it more clear, 
even more I suggest you remove it at all. 
please concentrate on finishing the important features be merged in main.

Reply:
Okay. I removed this section.